composer.json

```diff
@@ -39,7 +39,7 @@
         "illuminate/support": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8",
         "illuminate/validation": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8",
         "laragraph/utils": "^1",
-        "thecodingmachine/safe": "^1 || ^2",
```

Comment:
Hello @spawnia 

Was it intentional to remove the version 2 constraint?

Reply:
Oops, that was wrong. Good catch.

Reply:
Fantastic!! Thank you very much for the speedy release!